Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:22.10 as builder
+
+WORKDIR /root
+RUN apt-get -y update && apt-get install -y git cmake pkg-config libpcap-dev autoconf libtool
+
+RUN git clone https://github.com/utoni/nDPId.git
+#for dev, uncomment below
+#RUN mkdir /root/nDPId
+#COPY . /root/nDPId/
+
+RUN cd nDPId && mkdir build && cd build && cmake .. -DBUILD_NDPI=ON && make
+
+
+FROM ubuntu:22.10
+WORKDIR /root
+RUN apt-get -y update && apt-get -y install libpcap-dev
```

Comment:
```suggestion
RUN apt-get -y update && apt-get -y install libpcap
```

Reply:
it shall be libpcap0.8, libpcap can't be found